CitationStyleLanguagePlugin.php

```diff
@@ -493,12 +493,7 @@ public function getCitation(PKPRequest $request, Submission $submission, string
             throw new Exception('Unknown submission content type!');
         }
 
-        /** @var SubmissionLanguageDAO $submissionLanguageDao */
-        $submissionLanguageDao = DAORegistry::getDAO('SubmissionLanguageDAO');
-        $languages = $submissionLanguageDao->getLanguages($publication->getId(), [Locale::getLocale()]);
-        if (array_key_exists(Locale::getLocale(), $languages)) {
-            $citationData->languages = $languages[Locale::getLocale()];
-        }
+        $citationData->language = $submission->getData('locale');
```

Comment:
Shouldn't we use here the galley locales for `$languages`? -- I think this would be the same functionality as till now, no?